backend/wmg/data/schemas/cube_schema.py

```diff
@@ -33,7 +31,7 @@
 # The full set of logical cube dimensions by which the cube can be queried.
 expression_summary_logical_dims = expression_summary_indexed_dims + expression_summary_non_indexed_dims
 
-filters = [tiledb.ZstdFilter(level=+22)]
+filters = [tiledb.DictionaryFilter(), tiledb.ZstdFilter(level=+19)]
```

Comment:
applying the dictionary filter to dictionary-encode strings. this reduces cube size from 8gb to 2.7gb.

Reply:
1. why did you choose to greater compression? Typically, greater compression comes at the cost of query speed

2. What made you bump the compression level of Zstandard from 22 to 19?

Reply:
1. In this case, DictionaryFilter will speed up queries because it dictionary encodes bulky variable-length strings. Queries on integer arrays are much faster than queries on arrays with variable-length types.
2. A small performance optimization suggested by Bruce. Typically, the disk space savings you get with increasing levels of compression follow diminishing returns. I chose 19, but it could very well be 5. I haven't empirically searched across compression levels yet. In any case, 22 is overkill.